PhotoMapTests/ViewModels/TimelineViewModelTests.swift

```diff
@@ -0,0 +1,161 @@
+        firestoreService.userHasDocuments = true
+        firestoreService.markers = [Marker(category: "Nature", date: Date(), photoURLString: "url", location: (35, 89))]
+        
+        let expectation = XCTestExpectation()
+        
+        viewModel.reloadDataSubject.sink(receiveValue: { _ in
+            expectation.fulfill()
+        })
+        .store(in: cancelBag)
+        
+        XCTAssertFalse(firestoreService.getMarkersCalled)
+        viewModel.viewDidLoad()
+        wait(for: [expectation], timeout: 2)
+        XCTAssertTrue(firestoreService.getMarkersCalled)
+        XCTAssertTrue(firestoreService.getMarkersEndWithValues)
+        XCTAssertTrue(viewModel.numberOfSections > 0)
+    }
+    
+    func testIfGetValuesThenShouldGetTitlesForValues() {
+        firestoreService.userId = "id"
+        firestoreService.userHasDocuments = true
+        let markers = [
+            Marker(category: "Nature", date: Date(), description: "nature",
+                   hashtags: ["#nature"], photoURLString: "url", location: (15, 28))
+        ]
+        firestoreService.markers = markers
+        
+        let expectation = XCTestExpectation()
+        
+        viewModel.reloadDataSubject.sink(receiveValue: { _ in
+            expectation.fulfill()
+        })
+        .store(in: cancelBag)
+        
+        viewModel.viewDidLoad()
+        wait(for: [expectation], timeout: 2)
+        XCTAssertNotNil(viewModel.getTitle(for: markers.count - 1))
+    }
+    
+    func testIfGetValuesThenShouldGetChosenMarker() {
+        firestoreService.userId = "id"
+        firestoreService.userHasDocuments = true
+        let markers = [
+            Marker(category: "Nature", date: Date(), description: "nature",
+                   hashtags: ["#nature"], photoURLString: "url", location: (15, 28))
+        ]
+        firestoreService.markers = markers
+        
+        let expectation = XCTestExpectation()
+        
+        viewModel.reloadDataSubject.sink(receiveValue: { _ in
+            expectation.fulfill()
+        })
+        .store(in: cancelBag)
+        
+        viewModel.viewDidLoad()
+        wait(for: [expectation], timeout: 2)
+        let indexPath = IndexPath(row: markers.count - 1, section: 0)
+        XCTAssertNotNil(viewModel.getMarker(at: indexPath))
+    }
+    
+    func testIfGetValuesShouldReturnNumberOfRows() {
+        firestoreService.userId = "id"
+        firestoreService.userHasDocuments = true
+        let markers = [Marker(category: "Nature", date: Date(), photoURLString: "url", location: (35, 89))]
+        firestoreService.markers = markers
+        
+        let expectation = XCTestExpectation()
+        
+        viewModel.reloadDataSubject.sink(receiveValue: { _ in
+            expectation.fulfill()
+        })
+        .store(in: cancelBag)
+        
+        viewModel.viewDidLoad()
+        wait(for: [expectation], timeout: 2)
+        
+        XCTAssertTrue(viewModel.getNumberOfRows(in: markers.count - 1) > 0)
```

Comment:
Compare to expected value (e.g. you expect to see 2 rows with the passed markers).
By doing so it makes tests more accurate and precise